### PR TITLE
refactor: replace gql method addBulkItemsCart to addItemsCart in wishlist

### PR DESCRIPTION
**Pull request by NaMax66**
## Description
https://virtocommerce.atlassian.net/browse/ST-5201

### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.42.0-pr-823-2730-273091b7.zip

### CODE REVIEW

**Review by asvishnyakov — COMMENTED** [review]
`chore` is for periodical technical tasks, like packages & node update, lint rules updte, etc. Other types:
`feat`: new feature
`fix`: bug fix
`build`: changes in build
`ci`: changes in ci
`docs`: changes in docs
`perf`: performance improvement
`refactor`: refactoring
`style`: style changes
`test`:  changes in tests

https://www.conventionalcommits.org/en/v1.0.0/
https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type

So this is `fix` or `refactor`

**Comment by NaMax66** [comment]
> `chore` is for technical tasks which doesn't affect code or effect as side effect (node update, packages update, eslint update, storybook-related only task) `feat`: new feature `fix`: bug fix `build`: changes in build (vite, build scripts) `ci`: changes in ci (github actions) `docs`: changes in docs `perf`: performance improvement `refactor`: refactoring `style`: style changes `test`: changes in tests
> 
> https://www.conventionalcommits.org/en/v1.0.0/ https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type
> 
> So this is `fix` or `refactor`

refactor is ok
https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md

**Comment by asvishnyakov** [comment]
> https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md
It's a little bit outdated, we follow commitlint and conventional commits rules directly, so for example, we can use only lowercase letters in commits

**Comment by NaMax66** [comment]
> > https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md
> > It's a little bit outdated, we follow commitlint and conventional commits rules directly, so for example, we can use only lowercase letters in commits

https://github.com/VirtoCommerce/styleguide/issues/10

**Review by asvishnyakov — APPROVED** [review]

File: client-app/pages/account/list-details.vue [hunk 1]
```diff
@@ -305,6 +313,7 @@ async function canChangeRoute() {
 onBeforeRouteLeave(canChangeRoute);
 onBeforeRouteUpdate(canChangeRoute);
 
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
```
Comment (asvishnyakov) [inline client-app/pages/account/list-details.vue:316]: As discussed in dev chat, it's better to configure [no-misused-promises](https://typescript-eslint.io/rules/no-misused-promises/) to ignore such case
```json
{
  "@typescript-eslint/no-misused-promises": [
    "warn",
    {
      "checksVoidReturn": {
        "arguments": false,
      }
    }
  ]
}
```

**Pull request merged**